src/AccessibilityInsights.Extensions.AzureDevOps/ConfigurationControl.xaml

```diff
@@ -0,0 +1,176 @@
+﻿<!-- Copyright (c) Microsoft. All rights reserved.
```

Comment:
How can I view this control in the UI to check its accessibility?

Reply:
This is just the production azure dev ops connection control moved over to the extension, so its accessibility is hopefully unchanged. That being said, you should be able to find it in the connections tab of settings if you pull this branch. There were some other changes made to the tab which are not a part of this PR.